cmd/login_cloud.go

```diff
@@ -20,22 +20,24 @@ import (
 //nolint:funlen,gocognit
 func getCmdLoginCloud(gs *state.GlobalState) *cobra.Command {
 	// loginCloudCommand represents the 'login cloud' command
+	exampleText := getExampleText(gs, `
+  # Show the stored token.
+  {{.}} login cloud -s
+  
+  # Store a token.
+  {{.}} login cloud -t YOUR_TOKEN
+  
+  # Log in with an email/password.
+  {{.}} login cloud`[1:])
+
 	loginCloudCommand := &cobra.Command{
 		Use:   "cloud",
 		Short: "Authenticate with Load Impact",
 		Long: `Authenticate with Load Impact.
```

Comment:
Unrelated to this PR: we should update this, since we haven't been Load Impact for a few years now. Though I think we might deprecate this command soon, so it maybe doesn't matter.